test/functional/android/appium_service_tests.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
this is not needed anymore for python3

Reply:
I'll check and remove this line if possible. (not only here)